spring-aot-test/src/main/java/org/springframework/aot/test/context/bootstrap/generator/TestContextAotProcessor.java

```diff
@@ -126,6 +132,34 @@ protected ClassName generateTestContext(BootstrapWriterContext writerContext, Su
 				.addModifiers(Modifier.PUBLIC, Modifier.STATIC).addCode(code.build());
 	}
 
+	private MethodSpec.Builder getContextInitializersBuilder(Map<ClassName, TestContextConfigurationDescriptor> entries) {
+		// We're generating a method that looks like the following.
+		//
+		// public static Map<String, Class<? extends ApplicationContextInitializer<?>>> getContextInitializers() {
+		//     Map<String, Class<? extends ApplicationContextInitializer<?>>> map = new HashMap<>();
+		//     map.put("org.example.Sample1Tests", Sample1TestsContextInitializer.class);
+		//     map.put("org.example.Sample2Tests", Sample2TestsContextInitializer.class);
+		//     return map;
+		// }
+
+		ClassName stringTypeName = ClassName.get(String.class);
+		TypeName initializerWildcard = WildcardTypeName.subtypeOf(Object.class);
+		TypeName initializerTypeName = ParameterizedTypeName.get(ClassName.get(ApplicationContextInitializer.class), initializerWildcard);
+		TypeName classWildcard = WildcardTypeName.subtypeOf(initializerTypeName);
+		TypeName classTypeName = ParameterizedTypeName.get(ClassName.get(Class.class), classWildcard);
+		TypeName mapTypeName = ParameterizedTypeName.get(ClassName.get(Map.class), stringTypeName, classTypeName);
```

Comment:
That's a lot of code to get `Map<String, Class<? extends ApplicationContextInitializer<?>>>`.  The following single statement does the same thing: 

```java
TypeName mapType = ParameterizedTypeName.get(ClassName.get(Map.class),
				ClassName.get(String.class), ParameterizedTypeName.get(ClassName.get(Class.class), WildcardTypeName.subtypeOf(ApplicationContextInitializer.class)));
```

Reply:
The existing code produces:

```java
Map<String, Class<? extends ApplicationContextInitializer<?>>>
```

The proposed change produces the following which loses the `<?>`:

```java
Map<String, Class<? extends ApplicationContextInitializer>>
```

Although the proposed change is technically a single statement, it's 202 characters long, and I don't think it improves readability or maintainability of the code.

In any case, I'll see about reducing the number of local variables currently in use.

Reply:
Not going to comment any further on the 202 remark besides the fact that loosing variable assignment was the goal here. Sorry I missed the extra wildcard.